backend/app/views/spree/admin/reports/sales_total.html.erb

```diff
@@ -10,7 +10,7 @@
 <% end %>
 
 <% content_for :table_filter do %>
-  <%= render partial: 'spree/admin/shared/report_order_criteria', locals: {} %>
+  <%= render partial: 'spree/admin/shared/report_order_criteria', locals: { key: :sales_total } %>
```

Comment:
@tvdeyen What if I use `report_key` to be consistent with the controller https://github.com/solidusio/solidus/blob/master/backend/app/controllers/spree/admin/reports_controller.rb#L11 ? otherwise I think action is a good name too